src/routes/Ecosystem/Common/VaultField.tsx

```diff
@@ -3,32 +3,71 @@ import { Typography } from '@components/atomic';
 import { Box } from '@mui/system';
 import React from 'react';
 import { titleStyles } from './styles';
-import { ReactComponent as Mellow } from '../mellow-icon.svg';
+import { ReactComponent as USDC } from './usdc-icon.svg';
+import { ReactComponent as ETH } from './eth-icon.svg';
+import { ReactComponent as DAI } from './dai-icon.svg';
+import { ReactComponent as USDT } from './usdt-icon.svg';
+
+const getTokenIcon = (token: string) => {
```

Comment:
dont we already have these in the application? All you need to do is <Icon name="token-dai" /> iirc? Search for such usage. I don't think we need new set of icons but I might be wrong

Reply:
They have different backgrounds.

Reply:
I see thanks!